packages/atomic/src/components/result-template-components-v1/atomic-result-sections/atomic-result-section-visual.tsx

```diff
@@ -2,19 +2,18 @@ import {Element, Component} from '@stencil/core';
 import {containsVisualElement} from '../../../utils/utils';
 
 /**
- * The visual section helps the information seeker understand information about
- * the item visually, for example in Commerce an image is a great descriptive
- * for some category of products, an icon can help understand what type an item
- * is or an avatar understand whom it is related to.
+ * This section provides visual information about the item.
+ * For example, in Commerce, an image is a great shorthand for a product category.
+ * An icon can quickly show the item type, or an avatar can quickly show the target customer.
```

Comment:
I'm a bit confused by `target customer`, which seems more commerce-centric. I imagine that if we display an avatar, we mean for it to be in a list of people, which wouldn't really be for commerce.

Reply:
Good point! I will reconsider perspective on that.